rules/detekt/src/test/kotlin/com/twitter/compose/rules/detekt/ComposeModifierComposableCheckTest.kt

```diff
@@ -28,4 +29,16 @@ class ComposeModifierComposableCheckTest {
         assertThat(errors[0]).hasMessage(ComposeModifierComposable.ComposableModifier)
         assertThat(errors[1]).hasMessage(ComposeModifierComposable.ComposableModifier)
     }
+
+    @Test
+    internal fun `do not error on a regular composable`() {
```

Comment:
remove `internal`, for consistency with all the rest of tests

Reply:
Removed the internal keyword from the function